src/components/Menu/MenuItem.stories.tsx

```diff
@@ -24,7 +24,7 @@ import { MenuItem as MenuItemComponent } from "./MenuItem";
 import { Text } from "../Typography/Text";
 
 export default {
-  title: "MenuItem",
+  title: "Menu/MenuItem",
   component: MenuItemComponent,
   tags: ["autodocs"],
   argTypes: {},
```

Comment:
```suggestion
  argTypes: {},
  parameters: {
    design: {
      type: "figma",
      url: "https://www.figma.com/file/rTaQE2nIUSLav4Tg3nozq7/Compound-Web-Components?type=design&node-id=712-6909&mode=design&t=7zspBeYd4EBaDpzt-0",
    },
  },
```

Reply:
Hmm weird, they seem to work fine with latest storybook and I can't find anything about https://github.com/storybookjs/addon-designs support going away